source/subs_functions/subs_functions.bas

```diff
@@ -2833,11 +2833,11 @@ regid
 clearid
 id.n = "Cls"
 id.subfunc = 2
-id.callname = "sub_cls"
-id.args = 2
-id.arg = MKL$(LONGTYPE - ISPOINTER) + MKL$(ULONGTYPE - ISPOINTER)
-id.specialformat = "[?][,?]"
-id.hr_syntax = "CLS [method%] [, bgColor&]"
+id.callname = "sub_clsDest"
+id.args = 3
+id.arg = MKL$(LONGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER) + MKL$(ULONGTYPE - ISPOINTER)
+id.specialformat = "[?][,?][,?]"

```

Comment:
Shouldn't it be something like `[?][,[?][,?]]`?

Reply:
Sorry, my point was that the syntax `[?][,?][,?]` implies that you can include the third parameter without the comma for the second. The syntax `[?][,[?][,?]]` requires that to use the third argument you have to include the comma for the second.

Ex. [_LOADIMAGE](https://github.com/QB64-Phoenix-Edition/QB64pe/blob/e84c80e5ff511d205af661a2636782fd04a1f4b6/source/subs_functions/subs_functions.bas#L1066) with two optional parameters is written this way.